src/test/java/automation/examples/storefront/RunCucumberTest.java

```diff
@@ -1,14 +1,14 @@
-package automation.examples;
-
-import cucumber.api.CucumberOptions;
-import cucumber.api.junit.Cucumber;
-import org.junit.runner.RunWith;
-
-@RunWith(Cucumber.class)
-@CucumberOptions(
-        glue = "automation.examples.storefront.business",
-        features = "src/test/resources/features",
-        plugin = {"pretty", "json:target/cucumber-report/cucumber.json"}
-)
-public class RunCucumberTest {
-}
+package automation.examples;
+
+import cucumber.api.CucumberOptions;
+import cucumber.api.junit.Cucumber;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(
+        glue = "automation.examples.storefront.business",
+        features = "src/test/resources/features/storefront",
```

Comment:
Add the same changes to profile in pom pls

Reply:
@YuriiKononenko will add profiles in pom